frontend/src/features/moti/mapSideBar/containers/MotiInventoryContainer.tsx

```diff
@@ -1,27 +1,113 @@
+import { PARCELS_LAYER_URL, useLayerQuery } from 'components/maps/leaflet/LayerPopup';
 import MapSideBarLayout from 'features/mapSideBar/components/MapSideBarLayout';
-import { useQueryParamSideBar } from 'features/mapSideBar/hooks/useQueryParamSideBar';
-import { FormikValues } from 'formik';
-import { noop } from 'lodash';
+import {
+  SidebarContextType,
+  useQueryParamSideBar,
+} from 'features/mapSideBar/hooks/useQueryParamSideBar';
+import { FormikProps, FormikValues, getIn } from 'formik';
+import { FeatureCollection, GeoJsonProperties, Geometry } from 'geojson';
+import { useApiLtsa } from 'hooks/pims-api/useApiLtsa';
+import { IGeocoderResponse } from 'hooks/useApi';
+import { ParcelInfo } from 'interfaces/ltsaModels';
+import { LatLngLiteral } from 'leaflet';
 import * as React from 'react';
+import { toast } from 'react-toastify';
 
+import PropertyForm from '../forms/PropertyForm';
```

Comment:
Makes me think we should add an `index.ts` to the `../forms` path so we can pull in a multiple forms in a single line.

Reply:
Yes, I should get more in the habit of doing this. I'll add an index to my follow up PR.